templates/taxi/car_form.html

```diff
@@ -2,11 +2,11 @@
 {% load crispy_forms_filters %}
 
 {% block content %}
-  <h1>{{ object|yesno:"Update,Create" }} car</h1>
+  <h1>{{ object|yesno:"Update,Create" }} Car</h1>
   <form action="" method="post" novalidate>
     {% csrf_token %}
     {{ form|crispy }}
 
-    <input type="submit" value="Submit" class="btn btn-primary">
+    <input type="Submit" value="Submit" class="btn btn-primary">
```

Comment:
Consider changing `Submit` to `submit` for consistency and to follow HTML standards. 📝